clients/geth/specular/rollup/services/validator/validator.go

```diff
@@ -132,12 +132,15 @@ func (v *Validator) validationLoop(genesisRoot common.Hash) {
 			case errors.Is(err, errValidationFailed):
 				// Validation failed, challenge
 				isInChallenge = true
+				return nil
 			case errors.Is(err, errAssertionOverflowedInbox):
```

Comment:
i'd log something here at least (or before returning the err above). also consider changing the name (e.g. `errPendingSync`) since it's a bit unclear whether it's overflowing the L1 inbox or L2 blockchain current state